web/src/buddy_list.js

```diff
@@ -86,6 +88,77 @@ export class BuddyList extends BuddyListConf {
     all_user_ids = [];
     users_matching_view_ids = [];
     other_user_ids = [];
+    tooltips_initialized = false;
+
+    initialize_tooltips() {
+        $("#right-sidebar").on("mouseenter", ".buddy-list-heading", (e) => {
+            e.stopPropagation();
+            const $elem = $(e.currentTarget);
+            let placement = "left";
+            if (window.innerWidth < media_breakpoints_num.md) {
+                // On small devices display tooltips based on available space.
+                // This will default to "bottom" placement for this tooltip.
+                placement = "auto";
+            }
+            const subscriber_count_function = this.total_subscriber_count;
```

Comment:
Because `this` is different inside `onShow`. I've seen `that = this` things, but find that confusing. Is this a reasonable way of doing this?

Reply:
I think that's okay. It might read better to set the variable to `get_total_subscriber_count`, though.

Reply:
cool, did that